pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -102,7 +102,7 @@
 
   protected final AtomicLong _requestIdGenerator = new AtomicLong();
   protected final BrokerRequestOptimizer _brokerRequestOptimizer = new BrokerRequestOptimizer();
-  protected final BrokerReduceService _brokerReduceService = new BrokerReduceService();
+  protected BrokerReduceService _brokerReduceService;
```

Comment:
This can still be final?

Reply:
Yeah, not sure what happened there. Probably a side effect of trying out some intermediate code.